options.go

```diff
@@ -149,6 +149,9 @@ type grOptions struct {
 	GoroutineTriggerPercentDiff int // goroutine trigger diff in percent
 	GoroutineTriggerNumAbs      int // goroutine trigger abs in number
 
+	// if current goroutine number is greater than GoroutineNumberMax
+	// holmes would not dump goroutine profile.
+	GoroutineNumberMax int
```

Comment:
Maybe we can named `GoroutineTriggerNumMax` ,  and dump enable scope is `(minx ,max)`.
What's @cch123 's opinion?

Reply:
> Maybe we can name `GoroutineTriggerNumMax` , and dump trigger scope is `(min ,max)`. What's @cch123 's opinion?

Yes, this is better~